src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpOnTypeFormattingPass.cs

```diff
@@ -68,8 +68,21 @@ public async override Task<FormattingResult> ExecuteAsync(FormattingContext cont
                 var documentOptions = await GetDocumentOptionsAsync(context).ConfigureAwait(false);
 
                 // Ask C# for formatting changes.
+                var indentationOptions = new RazorIndentationOptions(
```

Comment:
@davidwengier would you mind confirming that this change looks correct here? I had to upgrade the Roslyn package version which resulted in one of the overloads being rendered obsolete. I was just going to leave it for later, but then the formatting tests started failing so I just attempted to fix it up in this PR😅

Reply:
I think if the tests pass then its fine, though @tmat used a different approach for the auto formatting options in https://github.com/dotnet/razor-tooling/pull/6166/files#diff-1fcf96ce7b4faaa3af72db165764d43a107a153f7040dc46169bd4d562f1b173 and got them from the global options service.

I'm not sure how much it matters for razor scenarios, as there isn't a way for anyone to configure the options anyway.

Reply:
Thanks! Didn't know Tomas opened a PR. I tried out his changes but was getting some weird casting errors when running the formatting tests. I'm going to take the current approach for now so we can merge this PR, but maybe we can address in a follow up

Reply:
:+1: His PR might depend on un-merged Razor EA changes in Roslyn.